packages/vue/src/stories/getting-started/tailwind-css.stories.mdx

```diff
@@ -0,0 +1,295 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+import { YouTube } from 'mdx-embed';
+
+<Meta title="Setup/Integrations/TailwindCSS" />
+
+# TailwindCSS
+
+[TailwindCSS](https://tailwindcss.com/) is a utility-first CSS framework packed with classes like flex, pt-4, text-center and rotate-90 that can be composed to build any design, directly in your markup.
+You can use the it to customize **StorefrontUI** components in your application.
+
+<YouTube youTubeId="uCxPb_39TNg" />
+
+## Vue 2 based projects
+
+* Install TailwindCSS packages
+
+```bash
+yarn add -D tailwindcss@npm:@tailwindcss/postcss7-compat postcss@^7 autoprefixer@^9
+```
+
+* Generate `tailwind.config.js`
+
+```bash
+npx tailwindcss init
+```
+
+This will generate basic config file
+
+```js
+// tailwind.config.js
+
+module.exports = {
+  purge: [],
+  darkMode: false, // or 'media' or 'class'
+  theme: {
+    extend: {},
+  },
+  variants: {
+    extend: {},
+  },
+  plugins: [],
+}
+```
+
+* Create index.css file with TailwindCSS styles and include it in main.js
+
+```bash
+cd src && touch index.css
+```
+
+```css
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
* Create index.css file  

```bash
cd src && touch index.css
```

* Import TailwindCSS styles inside

```css
@tailwind base;
@tailwind components;
@tailwind utilities;
```

* Next add them to main.js file